tests/foreman/ui/test_org.py

```diff
@@ -793,26 +808,28 @@ def test_add_computeresource(self, resource_name):
 
         """
         strategy, value = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        url = LIBVIRT_RESOURCE_URL % settings.server.hostname
-        resource = entities.LibvirtComputeResource(
-            name=resource_name,
-            url=url,
-        ).create()
-        self.assertEqual(resource.name, resource_name)
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name)
-            self.assertIsNotNone(self.org.search(org_name))
-            self.org.update(org_name, new_resources=[resource_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_resources'])
-            element = session.nav.wait_until_element((strategy,
-                                                      value % resource_name))
-            self.assertIsNotNone(element)
+            for resource_name in generate_strings_list():
```

Comment:
Same comment as before about using one single organization and adding entity to it and asserting its presence as well as increase of entities associated to the organization. Not a blocker for this PR.